tensorboard/webapp/feature_flag/actions/feature_flag_actions.ts

```diff
@@ -43,3 +43,16 @@ export const featureFlagOverrideChanged = createAction(
     flags: Partial<FeatureFlags>;
   }>()
 );
+
+export const featureFlagOverridesReset = createAction(
+  '[FEATURE FLAG] Resetting feature flag overrides',
+  props<{
+    flags: Array<keyof FeatureFlags>;
+  }>()
+);
+
+export const featureFlagsOverridesAllSetToDefault = createAction(
```

Comment:
What is the difference between "Reset" and "SetToDefault"? If they are the same thing, can we pick one name and use it consistently?

Reply:
+1

Reply:
Consistency in inconsistence